crates/bitwarden-vault/src/cipher/cipher_permissions.rs

```diff
@@ -0,0 +1,10 @@
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Copy, Deserialize, Debug, JsonSchema, Clone, PartialEq)]
+#[serde(rename_all = "camelCase", deny_unknown_fields)]
+#[cfg_attr(feature = "uniffi", derive(uniffi::Record))]
+pub struct CipherPermissions {
```

Comment:
I chatted with the [SDK folks](https://bitwarden.slack.com/archives/C054ZQSBS49/p1740764408393449), and we feel that we don't need a view for this model since it doesn't have any encrypted data, and the chance of it having any in the future is extremely low.